common_util/src/time.rs

```diff
@@ -56,6 +67,11 @@ pub fn current_time_millis() -> u64 {
     Utc::now().timestamp_millis() as u64
 }
 
+#[inline]
+pub fn current_time_secs() -> u64 {
```

Comment:
Remove this function.

In CeresDB, args/returns of time related fns are in ms unit.